DBTableToHBaseTable/README.rst

```diff
@@ -0,0 +1,57 @@
+Batch Database Table To CDAP HBase Table Adapter Configuration
```

Comment:
First line is an over-line of equal signs: see

https://raw.githubusercontent.com/cdap-guides/cdap-cube-guide/develop/README.rst